examples/flutter_gallery/ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -113,4 +113,4 @@
     "version" : 1,
     "author" : "xcode"
   }
-}
+}
```

Comment:
looks like the IDE made some unexpected changes here

Reply:
^^^^

Reply:
Fixed